### ISSUE TITLE
TFlite to 8 bit quantized TFLite conversion

### ISSUE THREAD

**Issue by sandeep22-v** [tensorflow/tensorflow#48637]
I have a TFLite model with me sans any pb model and it is operating on float32 (all the parameters including weights,biases and activaions).
I need to get the 8 bit integer quantized version of this.
Is there any way in which I can convert TFlite model to 8 bit quantized TFLite model without having pb file of it??

**Comment by abattery** [comment]
Please refer to https://www.tensorflow.org/lite/performance/post_training_quantization#full_integer_quantization

**Comment by sandeep22-v** [comment]
> Please refer to https://www.tensorflow.org/lite/performance/post_training_quantization#full_integer_quantization

Thanks for the reply sir!!

I have checked the website mentioned by you but the issue I faced there was there the tflite model is generated from a saved_model ,i.e,  we need to input a pb model ,then convert it to tflite model

converter = tf.lite.TFLiteConverter.from_saved_model(saved_model_dir)


But my requirement is a bit different. I need to convert an already converted tflite model to  8 bit quantized tflite model.

Is there anyway in which we can do that sir?
